app/src/main/java/org/mifos/mobile/ui/activities/PrivacyPolicyActivity.kt

```diff
@@ -7,52 +7,44 @@ import android.os.Bundle
 import android.view.View
 import android.webkit.WebView
 import android.webkit.WebViewClient
-import android.widget.ProgressBar
 
-import butterknife.BindView
-import butterknife.ButterKnife
 
 import org.mifos.mobile.R
+import org.mifos.mobile.databinding.ActivityPrivacyPolicyBinding
 import org.mifos.mobile.ui.activities.base.BaseActivity
 
 /**
  * @author Rajan Maurya
  * On 11/03/19.
  */
 class PrivacyPolicyActivity : BaseActivity() {
-
-    @JvmField
-    @BindView(R.id.webView)
-    var webView: WebView? = null
-
-    @JvmField
-    @BindView(R.id.progress_bar)
-    var progressBar: ProgressBar? = null
+    private lateinit var binding : ActivityPrivacyPolicyBinding
     private var showOrHideWebViewInitialUse = "show"
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_privacy_policy)
-        ButterKnife.bind(this)
+        binding = ActivityPrivacyPolicyBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
```

Comment:
this is perfect but I would suggest summing them up in a single line . Not that it would make much of a difference
```suggestion
        val view = binding.root
        setContentView(view)
```

Reply:
So something like this, 
setContentView(binding.root), right?

Reply:
yes !!